src/FirebirdSql.EntityFrameworkCore.Firebird.Tests/Scaffolding/ScaffoldingTests.cs

```diff
@@ -15,16 +15,37 @@
 
 //$Authors = Jiri Cincura (jiri@cincura.net)
 
+using System;
 using System.Linq;
 using System.Threading.Tasks;
+using FirebirdSql.Data.FirebirdClient;
+using FirebirdSql.Data.Services;
+using FirebirdSql.EntityFrameworkCore.Firebird.Metadata;
+using FirebirdSql.EntityFrameworkCore.Firebird.Metadata.Internal;
 using FirebirdSql.EntityFrameworkCore.Firebird.Scaffolding.Internal;
 using Microsoft.EntityFrameworkCore.Scaffolding;
+using Microsoft.EntityFrameworkCore.Scaffolding.Metadata;
 using NUnit.Framework;
 
 namespace FirebirdSql.EntityFrameworkCore.Firebird.Tests.Scaffolding;
 #pragma warning disable EF1001
 public class ScaffoldingTests : EntityFrameworkCoreTestsBase
 {
+	private DatabaseModel _databaseModel;
+	private Version _serverVersion;
+
+	public override async Task SetUp()
+	{
+		await base.SetUp();
+
+		_serverVersion = FbServerProperties.ParseServerVersion(Connection.ServerVersion);
+
+		await CreateScaffoldingObjectsAsync();
+
+		var modelFactory = GetModelFactory();
+		_databaseModel = modelFactory.Create(Connection, new DatabaseModelFactoryOptions());
```

Comment:
This could throw and it is what `JustCanRun` is testing. Better to get the model in each individual test.